### PR TITLE
Fixing typo in README.md

### PR DESCRIPTION
**Pull request by aik099**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Contribute
 Contributions to the documentation are always welcome!
 
 * Report any bugs or issues you find on the [issue tracker](https://github.com/minkphp/docs/issues).
-* You can grab the source code at Puli’s [Git repository](https://github.com/minkphp/docs).
+* You can grab the source code at Pull’s [Git repository](https://github.com/minkphp/docs).
```
Comment (stof) [inline README.md]: Well, actually Puli is a bad copy-paste (I copy-pasted the Puli documentation readme to create this one). Pull does not make sense here.

Comment (aik099) [inline README.md]: So `Puli` is name of a person? I don't get should it be corrected or not?

Comment (stof) [inline README.md]: it is the name of a project, from where I copy-pasted the Readme instead of starting from an empty file: https://github.com/puli/

Comment (stof) [inline README.md]: so it should be corrected, but not in this way (``Pull's`` makes less sense than the current wrong sentence)

Comment (aik099) [inline README.md]: OK, then I'm closing this PR.

**Pull request closed without merging**